packages/frontend/src/components/dialogs/AddMember/AddMemberDialog.tsx

```diff
@@ -32,6 +32,10 @@ export function AddMemberDialog({
     queryStrIsValidEmail,
     refreshContacts,
   } = useLazyLoadedContacts(listFlags, queryStr)
+
+  // compare bitwise if address flag is set
+  const allowAddManually = (listFlags & C.DC_GCL_ADDRESS) !== 0
```

Comment:
Hmmm, I'd prefer a more direct way to determine this.

Reply:
We could an additional parameter. I heard that there are anyway changes planned in that area so we will soon refactor here